.gitignore

```diff
@@ -17,5 +17,6 @@ build.sh
 publish.sh
 dist/*
 !dist/rudder-sdk-js/*
+dist/rudder-sdk-js/index.js
```

Comment:
[Non-blocking]
PR description needs to be updated.

Reply:
done